src/components/expanded-state/ExpandedStateSection.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-import styled from 'styled-components';
 import { ColumnWithMargins } from '../layout';
 import { Text } from '../text';
 import { padding } from '@rainbow-me/styles';
+import styled from 'rainbowed-components';
 
 const Container = styled(ColumnWithMargins).attrs({
   margin: 12,
-})`
-  ${padding(android ? 19 : 36, 19, 24)};
-  ${({ isNft }) => padding(android ? 19 : 36, isNft ? 24 : 19, 24)};
-  padding-top: ${({ isL2, isNft }) => (isL2 || isNft ? 24 : android ? 19 : 36)};
-`;
+})(({ isNft, isL2 }) => ({
+  ...padding.object(android ? 19 : 36, 19, 24),
+  paddingTop: isL2 || isNft ? 24 : android ? 19 : 36,
```

Comment:
This line was last in the string version, so looks like it needs to be moved down one line here to maintain the same precedence.